ui/pages/import-token/import-token.component.js

```diff
@@ -208,8 +215,22 @@ class ImportToken extends Component {
 
     const isMainnetNetwork = this.props.chainId === '0x1';
 
+    let standard;
+    if (addressIsValid) {
+      try {
+        ({ standard } = await this.props.getTokenStandardAndDetails(
+          standardAddress,
+          this.props.selectedAddress,
+        ));
+      } catch (error) {
+        // ignore
+      }
```

Comment:
should we add a log or may be a metrics event in catch block ?

Reply:
Yes good call! @brad-decker any thoughts on what the metric event could look like here?

Reply:
@jpuri I met with Kevin, Brad and Dan Miller today to discuss NFT tracking events and this is one that was raised. I think we need to define a few things for the schema first so I am intending to batch all the NFT tracking work in an upcoming PR, but if we can hold off on it for this PR that would be great.